### PR TITLE
Extra menubar buttons

### PR DESCRIPTION
**Pull request by kwendel**
Have a look at our new help menu!
Also added shortkey R to reset the view and buttons to move left and right in the view

### CODE REVIEW

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 1]
```diff
@@ -15,6 +36,38 @@
 	private Application application;
 
 	/**
+	 * Text that is displayed in the About Me option in the Help menu.
+	 */
+	final String about = "Helix\u00B2 is a interactive DNA sequence viewer. "
+			+ "It uses semantic zooming to only display relative information. \n"
+			+ "This application was created for as part of a assignment"
+			+ "for Contextproject on the TU Delft.\n"
+			+ "\n"
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: Static modifier here?

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 2]
```diff
@@ -15,6 +36,38 @@
...
+			+ "The code of this application is opensource and can be found on GitHub: \n";
+
+	/**
+	 * Text that is displayed in the Controls option in the Help menu.
+	 */
+	final String controls = "Helix\u00B2 uses keys shortcut to make life easier. "
+			+ "All the controls that can be used are listed below. \n"
+			+ "\n"
+			+ "Zooming in     \t+ \n"
+			+ "Zooming out    \t - \n"
```
Comment (Delfil) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: Static modifier here?

**Comment by Delfil** [comment]
Looks great!

**Comment by mathieupost** [comment]
Ziet dr lekker uit hoor

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 3]
```diff
@@ -15,6 +36,38 @@
 	private Application application;
 
 	/**
+	 * Text that is displayed in the About Me option in the Help menu.
+	 */
+	final String about = "Helix\u00B2 is a interactive DNA sequence viewer. "
+			+ "It uses semantic zooming to only display relative information. \n"
+			+ "This application was created for as part of a assignment"
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: Should be: "This application was created as part of an assignment"

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 4]
```diff
@@ -15,6 +36,38 @@
...
+			+ "for Contextproject on the TU Delft.\n"
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: for the Contextproject

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 5]
```diff
@@ -15,6 +36,38 @@
...
+			+ "\n"
+			+ "Helix\u00B2 was created by: \n"
+			+ "- Tom Brouws\n"
+			+ "- Boris Mattijssen\n"
+			+ "- Mathieu Post\n"
+			+ "- Sam Smulders\n"
+			+ "- Kasper Wendel\n"
+			+ "\n"
+			+ "The code of this application is opensource and can be found on GitHub: \n";
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: open source

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 6]
```diff
@@ -15,6 +36,38 @@
...
+
+	/**
+	 * Text that is displayed in the Controls option in the Help menu.
+	 */
+	final String controls = "Helix\u00B2 uses keys shortcut to make life easier. "
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: key shortcuts

File: src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java [hunk 7]
```diff
@@ -15,6 +36,38 @@
...
+			+ "All the controls that can be used are listed below. \n"
+			+ "\n"
+			+ "Zooming in     \t+ \n"
+			+ "Zooming out    \t - \n"
+			+ "Reset the view \t R \n"
+			+ "Move the view to the left \t left arrow \n"
+			+ "Move the view to the right \t right arrow \n"
+			+ "\n"
+			+ "All of the menu's can be controlled with the underlined letter, "
```
Comment (TomBrouws) [inline src/main/java/tudelft/ti2806/pl3/menubar/MenuBarController.java]: menus

**Comment by borismattijssen** [comment]
about_me_en_dna_bazen_v0_5

Looks a bit weird. The rest is all fine :)

**Comment by borismattijssen** [comment]
Looks good now!

**Pull request merged**